test/test.py

```diff
@@ -28,21 +28,23 @@
 >>> input_data["level"] == 1
 True
 >>> input_data = get_input_standard(lambda x: "1")
+- Please choose a program :
+<BLANKLINE>
+1- Clear Mind (~ 07 minutes)
+2- Relax1 (~ 07 minutes)
+3- Relax2 (~ 03 minutes)
+4- Relax3 (~ 06 minutes)
+5- Calming (~ 09 minutes)
+6- Power (~ 07 minutes)
+7- Harmony (~ 07 minutes)
+8- Anti-Stress (~ 04 minutes)
+9- Anti-Appetite (~ 10 minutes)
+10- Cigarette Replace (~ 05 minutes)
```

Comment:
I think it'd be better if we use non-format form for numbers, like:
```
1- Clear Mind (~ 7 minutes)
2- Relax1 (~ 7 minutes)
...
```

Reply:
Good point!
I will fix it ASAP 🚀